web/src/compose_recipient.ts

```diff
@@ -461,10 +458,16 @@ export function update_topic_displayed_text(topic_name = "", has_topic_focus = f
     } else {
         $topic_not_mandatory_placeholder.show();
         update_placeholder_visibility();
-        $input.on("input", update_placeholder_visibility);
```

Comment:
Yikes, I think this old code would stack up multiple copies of this handler, since it's not `.one()`...

But I'm confused by this commit's impact; now we're calling `update_placeholder_visibility` regardless of the branch? Maybe that's fixed in a following commit, I'll keep reading.

Reply:
Correct. I should probably improve that somehow, or at least comment about it, but `update_placeholder_visibility()` is visually a noop when topics are required.